### PR TITLE
Update guideline-09.md: legal compliance

### PR DESCRIPTION
**Pull request by webdevlaw**
Proposed addition: **implying that a plugin can create, provide, automate, or guarantee legal compliance**

Over the years there have occasionally been plugins launched which have claimed to provide legal compliance (privacy, data protection, contracts, copyright, etc) either in its description, its title, or in its creators' promotion of the work. However, no plugin can provide legal compliance on any issue on its own. While compliance-focused plugins can automate certain processes, provide workflows for complex obligations, or support administrators on a wider compliance journey, they cannot create any legal position which would protect a site administrator, much less provide the confidence that users expect from the sites they use.

Allowing plugins to claim they can provide "compliance" is very dangerous. It casts the ecosystem as a place where administrators can essentially outsource their legal obligations to seemingly random developers with no regard for accuracy or credentials. It also risks positioning the ecosystem as a marketplace which allows these claims to exist in the first place, inviting hostile public scrutiny - and perhaps rightfully so.

For these reasons, I would like to see "implying that a plugin can create, provide, automate, or guarantee legal compliance" added to the list of illegal, dishonest, and morally offensive claims.

### CODE REVIEW

**Comment by Ipstenu** [comment]
I don't have a problem with this at all. 👍 

In order to do this right I need to:

1) Get a list of all plugins that currently claim compliance (running a scan now... my laptop sounds like a jet plane)
2) Draft up a blog post that summarizes _why_ we're doing this
3) Once posted, email everyone who may be isshy about this in their plugins and explain why they need to change ASAP.

Here's the idea for the blog post (Heather, I've trimmed down what you said, so PLEASE tell me if I simplified too much!):

Legal Compliance Added to Guidelines

Guideline 9 (Developers and their plugins must not do anything illegal, dishonest, or morally offensive.) has been amended to include the following new prohibition:

> implying that a plugin can create, provide, automate, or guarantee legal compliance

This can be confusing, as the majority of plugins will never run into this and thus it's an entirely new concept to many people. 

The issue is that developers have, by accident or intent, claimed their plugins can provide legal compliance in various arenas. This includes security ('use this plugin and you will be FIOS compliant!'), Cookie Tracking, online shopping (such as for VAT compliance), GDPR, and so on.

Sadly, a plugin in and of itself _cannot_ actually provide any legal compliance. While a plugin can assist in automating the steps needed to be compliant, or allow you to develop a workflow to solve the situation, they cannot protect a site administrator from mistakes or lack of compliance.

In short, it's great the plugins are helping you, but we need to make sure they don't give people a false sense of security. 

Because of that, we will be monitoring plugins going forward and attempting to prevent these types of claims. This will be handled in the same way we try to make sure that people don't use 'official plugin' without actually being official. Plugins that are are currently at odds with this change will be notified shortly and required to change their readmes.

**Comment by Ipstenu** [comment]
Note: First scan has a lot of false positives. 1000 or so to review. 🙉

**Comment by Ipstenu** [comment]
Solid. I've made a draft post. I'm also working on the (ouch) longer email that will be sent out: https://gist.github.com/Ipstenu/96e9df825e30df15c351a9700bea33a7

And reviewing the 1000 or so hits. A _LOT_ are things like "Works with any standards compliant WordPress theme" so this is fun 😆

**Comment by Ipstenu** [comment]
```
241 matching plugins
Matches  Plugin                                                                    Active installs
=======  ======                                                                    ===============
      1  adblock-notify-by-bweb                                                            10,000+
      1  add-cookie-notice                                                                  2,000+
      1  advanced-cart-recovery                                                               200+
      1  advanced-custom-fields-recaptcha-field                                               600+
      1  ageverify-by-inverite                                                                 10+
      1  antispam-bee                                                                     300,000+
      1  appointment-hour-booking                                                              90+
      1  assembly-payments-gateways                                                             0+
      1  b2i-investor-tools                                                                    20+
      1  bancomail-email-lists-integration                                                     30+
      1  billdu-connector-for-woocommerce                                                       0+
      1  bitmovin-video                                                                        70+
      1  blue-utopia                                                                           10+
      1  bookingcom-affiliate-plugin                                                        1,000+
      1  browserid                                                                             70+
      1  brozzme-cookie-notification                                                          300+
      1  buy-now-plus                                                                          60+
      1  cardstream-payment-gateway                                                            30+
      1  cart66-cloud                                                                         800+
      1  cartograf-cookie-filter                                                            1,000+
      1  chargebee                                                                            100+
      1  circupress                                                                            20+
      1  civist                                                                                60+
      1  clearent-payments                                                                     40+
      1  codemonkeys-hipaa-forms                                                               10+
      1  cognito-forms                                                                      3,000+
      1  comment-mail                                                                         500+
      1  conekta-payment-gateway                                                              600+
      1  content-shelf-shopping-cart                                                           10+
      1  cookie-bar                                                                        10,000+
      1  cookie-law-bar                                                                     8,000+
      1  cookie-law-info                                                                  200,000+
      1  cookie-notice                                                                    500,000+
      1  cookie-ok                                                                            100+
      1  cookie-optin-interface                                                                 0+
      1  cookiebot                                                                            500+
      1  cookiecert-eu-cookie-directive                                                       100+
      1  creare-eu-cookie-law-banner                                                       10,000+
      1  custom-cookie-message                                                                100+
      1  debt-reduction-calculator-debt-relief-program-calculator-all-in-one                   90+
      1  digiproveblog                                                                      4,000+
      1  digital-signature-contact-form-7-addon                                               100+
      1  disclaimer-by-elan42                                                                 100+
      1  dsidxpress                                                                         9,000+
      1  duo-wordpress                                                                      7,000+
      1  easy-cookies-policy                                                                  500+
      1  easy-digital-downloads-https                                                          30+
      1  easy-donation                                                                        200+
      1  easy-review-reminders                                                                400+
      1  ebanx-payment-gateway-for-woocommerce                                                100+
      1  ebyline-payments                                                                      10+
      1  ecwid-shopping-cart                                                               20,000+
      1  edd-digital-signature-add-on                                                         100+
      1  edd-quaderno                                                                          10+
      1  electric-studio-eu-cookie-law-compliance                                             100+
      1  eshop-quantumwebform                                                                   0+
      1  eu-cookie-consent                                                                    300+
      1  eu-cookie-law-compliance                                                           4,000+
      1  eu-cookie-law-wp-cookie-law                                                          200+
      1  eu-cookie-notice                                                                     600+
      1  eu-opt-in-compliance-for-mailchimp                                                   400+
      1  eu-vat-redirect                                                                       10+
      1  eucookielaw                                                                        1,000+
      1  exit-notifier                                                                        900+
      1  expire-passwords                                                                   1,000+
      1  fanbridge-signup                                                                     300+
      1  fanbridge-toolbox                                                                     60+
      1  fmtc-affiliate-disclosure                                                          1,000+
      1  formidable                                                                       200,000+
      1  forms-signature-formidable-online-contract-automation                                 60+
      1  frankencookie                                                                         40+
      1  free-my-menu-embed                                                                    10+
      1  fs-revenue-mazimizer                                                                  10+
      1  ga-germanized                                                                        100+
      1  gateway-stripe                                                                        30+
      1  gdgt-databox                                                                          10+
      1  gdpr-banner                                                                           40+
      1  gdpr-compliance                                                                       20+
      1  ginger                                                                             7,000+
      1  google-adsense-for-responsive-design-gard                                          1,000+
      1  graceful-email-obfuscation                                                           600+
      1  gravity-forms-braintree                                                              100+
      1  gravity-forms-stripe                                                               2,000+
      1  gravity-signature-forms-add-on                                                     1,000+
      1  guestfront                                                                            20+
      1  gwolle-gb                                                                         40,000+
      1  hayona-cookies                                                                       100+
      1  helcim-commerce-for-woocommerce                                                       60+
      1  i-comply-social-media-sharing-protection-compliance-archiving-and-workflow-ap          0+
      1  identity-verification-management                                                      10+
      1  idx-broker-platinum                                                                9,000+
      1  ilmenite-cookie-consent                                                            1,000+
      1  italy-cookie-choices                                                              40,000+
      1  jabber-feed                                                                            0+
      1  jackmail-newsletters                                                               1,000+
      1  jigoshop-payflow-gateway                                                               0+
      1  jituzu-tools                                                                          90+
      1  jkl-reviews                                                                           50+
      1  library                                                                               60+
      1  local-market-explorer                                                              1,000+
      1  lockr                                                                                 10+
      1  mailjet-for-wordpress                                                             10,000+
      1  makecommerce                                                                         700+
      1  mangopay-woocommerce                                                                 200+
      1  manual-credit-card-processing-for-woocommerce                                         80+
      1  mashsharer                                                                        80,000+
      1  memberfindme                                                                       1,000+
      1  merchant-e-solutions                                                                  30+
      1  mijireh-checkout-for-ithemes-exchange                                                  0+
      1  mijireh-checkout-for-jigoshop                                                          0+
      1  miniorange-saml-20-single-sign-on                                                  1,000+
      1  mollie-payments-for-woocommerce                                                   30,000+
      1  mypos-virtual-for-woocommerce                                                        400+
      1  newsletter-manager                                                                 9,000+
      1  newsletter                                                                       300,000+
      1  nif-num-de-contribuinte-portugues-for-woocommerce                                  1,000+
      1  ninja-signature-contract-forms-add-on                                                400+
      1  oauth-proxy-service                                                                    0+
      1  oik-privacy-policy                                                                 4,000+
      1  oik                                                                                7,000+
      1  omnicard-lite                                                                         60+
      1  omnikassa                                                                            100+
      1  orderstorm-e-commerce                                                                 10+
      1  orderstorm-wordpress-e-commerce                                                       20+
      1  participants-database                                                             10,000+
      1  patient-education-h1n1-flu-tutorial                                                   10+
      1  paycorp-sampath-bank-ipg                                                               0+
      1  payiq-wc-gateway                                                                      10+
      1  payitlater-gateway-for-woocommerce                                                     0+
      1  payment-form-for-paypal-pro                                                          200+
      1  paymentsense-gateway-for-woocommerce                                                  20+
      1  paymill                                                                              800+
      1  phunnl                                                                                 0+
      1  piwik-pro-tag-manager-integration                                                     10+
      1  placester                                                                          1,000+
      1  policy-genius                                                                        300+
      1  post-to-email                                                                         80+
      1  power-forms-builder                                                                    0+
      1  privacy-cookie-law                                                                    60+
      1  privacybuilder                                                                         0+
      1  quaderno-checkout                                                                     70+
      1  quick-contact-form                                                                 8,000+
      1  quick-paypal-payments                                                              5,000+
      1  real-estate-mls-search                                                               100+
      1  realty-idx-pro                                                                       100+
      1  redbrick-digital-core                                                                 50+
      1  s-crib-otp-authentication                                                             20+
      1  saf-moss                                                                               0+
      1  sagepay-form-gateway-for-woocommerce                                                 500+
      1  screen-reader-with-fontsize                                                          100+
      1  securimage-wp                                                                      1,000+
      1  shareaholic                                                                      100,000+
      1  shariff                                                                           20,000+
      1  showcase-idx                                                                       1,000+
      1  signature-caldera-forms-online-contract-add-on                                        60+
      1  simple-cookie-law                                                                      0+
      1  simply-rets                                                                          300+
      1  single-sign-on-sso                                                                     0+
      1  snapskout                                                                            300+
      1  soldpress                                                                            400+
      1  splitit-installment-payments-enabler                                                  50+
      1  standout-color-boxes-and-buttons                                                   5,000+
      1  statify                                                                           30,000+
      1  stripe-gateway-for-events-manager-pro                                                300+
      1  stripe-political-donations                                                            10+
      1  stripe-woocommerce-addon                                                           7,000+
      1  super-stripe                                                                         300+
      1  surbma-gdpr-proof-google-analytics                                                     0+
      1  surbma-yes-no-popup                                                                  400+
      1  swaypay-woocommerce                                                                    0+
      1  swipezoom-global-payment-and-shipping                                                  0+
      1  taxify-for-woocommerce                                                                60+
      1  taxjar-simplified-taxes-for-woocommerce                                            4,000+
      1  telemed-payments                                                                       0+
      1  traffic-manager                                                                    3,000+
      1  transactium-gravity-forms-addon                                                        0+
      1  transactium-wallet-for-woocommerce                                                    10+
      1  transactium-woocommerce-addon                                                         10+
      1  turnstile                                                                              0+
      1  twispay                                                                               30+
      1  two-factor-authentication                                                          7,000+
      1  typekit-fonts-for-wordpress                                                       30,000+
      1  uk-cookie-consent                                                                100,000+
      1  uk-food-hygiene-rating                                                                10+
      1  uleak-security-dashboard                                                              30+
      1  ultimate-newsletter                                                                  200+
      1  ultracart-ecommerce-shopping-cart                                                     50+
      1  vat-ecsl                                                                               0+
      1  vat-moss                                                                              30+
      1  wc-azul-payment-gateway                                                               30+
      1  web-payment-software-payment-gateway-for-woocommerce                                  30+
      1  webba-booking-lite                                                                   900+
      1  webreserv-booking-calender-plugin                                                    500+
      1  wf-cookie-consent                                                                 20,000+
      1  windsor-zendesk-submit-ticket-form                                                    10+
      1  woo-adcoin-gateway                                                                     0+
      1  woo-authorize-net-gateway-aim                                                        100+
      1  woo-cloudswipe-payments                                                              100+
      1  woo-easy-pay                                                                         600+
      1  woo-fiscalita-italiana                                                               100+
      1  woo-forte-payment-gateway                                                             40+
      1  woo-gdpr-compliance                                                                    0+
      1  woo-moip-official                                                                    400+
      1  woo-myghpay-payment-gateway                                                           50+
      1  woo-mynix-braintree                                                                   50+
      1  woo-payment-gateway                                                                  500+
      1  woocommerce-digital-signature                                                        500+
      1  woocommerce-eu-vat-assistant                                                       2,000+
      1  woocommerce-eu-vat-compliance                                                      3,000+
      1  woocommerce-gateway-paypal-powered-by-braintree                                  100,000+
      1  woocommerce-germanized                                                            30,000+
      1  woocommerce-mailchimp-casl                                                            10+
      1  woocommerce-mijireh-checkout                                                       3,000+
      1  woocommerce-nochex-payment-gateway                                                   100+
      1  woocommerce-novalnet-gateway                                                         900+
      1  woocommerce-payment-gateway-ccbill                                                   400+
      1  woocommerce-payment-gateway                                                        1,000+
      1  woocommerce-quaderno                                                                  90+
      1  wordpress-dashboard-twitter                                                          600+
      1  wordpress-ecommerce                                                                6,000+
      1  wp-accessible                                                                        100+
      1  wp-ada-compliance-check-basic                                                         70+
      1  wp-business-intelligence-lite                                                        700+
      1  wp-cookiecuttr-by-nimrodstu                                                           70+
      1  wp-forms-signature-contract-add-on                                                   200+
      1  wp-gdpr-compliance                                                                 6,000+
      1  wp-gdpr-core                                                                         800+
      1  wp-gdpr-data-protection                                                                0+
      1  wp-gdpr-ultimate                                                                       0+
      1  wp-listings                                                                        7,000+
      1  wp-mailup                                                                          4,000+
      1  wp-payeezy-pay                                                                       900+
      1  wp-portfolio                                                                       3,000+
      1  wp-security-txt                                                                        0+
      1  wpsecurity                                                                         2,000+
      1  wsspg-woocommerce-stripe-subscription-payment-gateway                                800+
      1  xola-bookings-for-tours-activities                                                    10+
      1  yellow-schedule                                                                       70+
      1  yith-woocommerce-eu-vat                                                              100+
      1  yith-woocommerce-stripe                                                            3,000+
      1  zodiacpress                                                                          300+
```
